snowshu/core/utils.py

```diff
@@ -4,17 +4,16 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Optional, TextIO, Type, Union
 
+import logging
 import yaml
 
-from snowshu.logger import Logger
-
 if TYPE_CHECKING:
     from snowshu.adapters.base_sql_adapter import BaseSQLAdapter
     from snowshu.adapters.source_adapters.base_source_adapter import BaseSourceAdapter
     from snowshu.adapters.target_adapters.base_target_adapter import BaseTargetAdapter
 
 
-logger = Logger().logger
+logger = logging.getLogger(__name__)
```

Comment:
it will be represented as snowshu.adapters.<name> (the root logger is snowshu) - as expected